site/management.md

```diff
@@ -401,24 +402,20 @@ In addition to the `connect-src` CSP header, RabbitMQ also needs the CSP directi
 
 By default, the RabbitMQ Management UI uses the OAuth 2.0 **authorization code flow** to authenticate and authorize users.
 However, there are scenarios where users preferred to be automatically redirected to RabbitMQ without getting
-involved in additional logon flows. This is common in Web Portals where with a single click, users navigate
-straight to a RabbitMQ cluster's management UI with a token obtained under the covers. This is known as
+involved in additional logon flows. This is common in Oauth2 proxies and in Web Portals where with a single click, users navigate straight to a RabbitMQ cluster's management UI with a token obtained under the covers. This is known as
 **Identity-Provider initiated logon**.
 
```

Comment:
Hi @MarcialRosales first sentence in this section (text not in this review), the word "preferred" should be "prefer"

So reading the entire content above are we trying to say here that customers don't want to be involved in additional logon flows and that OAuth2 proxies and web portals help to avoid this - if this is what we are trying to say, I would rephrase the above to: 

_However, there are scenarios where users prefer to be automatically redirected to RabbitMQ without getting involved in additional logon flows. By using OAuth2 proxies and web portals, these additional logon flows can be avoided. With a single click, users navigate straight to a RabbitMQ cluster's management UI with a token obtained under the covers. This is known as **Identity-Provider initiated logon**._



Does "Web Portals" need to be in uppercase, if not I would change it to "web portals" mid sentence 

I think "RabbitMQ cluster's management UI" should be "RabbitMQ cluster management UI", the use of the word "a" before this makes the instance of "RabbitMQ cluster management UI" singular or plural which is what you are doing here

Also should "management UI" be in uppercase "Management UI" when I scroll up I see "Management UI" I guess use one or the other but keep it consistent throughout